recipes/pink/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pink" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/pin-{{ name }}/pin_{{ name }}-{{ version }}.tar.gz
+  sha256: daa245925c739d69a14ce2c4b80f9f3758648b7c6f4987b392a78692e6bc67f1
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - flit-core >=2,<4
+    - pip
+  run:
+    - python >=3.8
+    - loop-rate-limiters >=0.1.0
+    - numpy >=1.19.0
+    - osqp
+    - pinocchio >=2.6.3
+    - proxsuite
+    - qpsolvers >=2.5.0
+    - quadprog >=0.1.11
+
+test:
+  imports:
+    - pink
+  commands:
+    - python -c "import pinocchio as pin; print(pin)"
+    - python -c "import pink; print(pink)"
```

Comment:
You probably don't need these. One is testing a dependency and the other one is redundant with the test import above.

```suggestion
```

Reply:
Got it, thank you!